dist/tools/print_versions.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+echo "Environment Variables"
+echo "---------------------"
+printenv
```

Comment:
just tested, this could introduce a security risk by disclosing system infos to the public. I think we should rather explicitly name the variable we want to print.

Reply:
variables would be BOARD, PHILIP_PORT, DEBUG pins and ports, and so on. Should look similar to the RIOT script, in explicit should write `missing` if variable is not set?!

Reply:
I would be interested in more than just that though...  I am guessing you would insist on that?  I guess I am just thinking we would miss some useful information somewhere.

Reply:
Or could we also add an environment namespace.  For example make everything that uses robot framework tests prefix with `RF_*`, we can export that along with the standard `BOARD, PHILIP_PORT, ESP8266_RTOS_SDK_DIR, PATH, PYTHONPATH, UNIFLASH_PATH, etc)?

Reply:
Would we also want to try to get what comes from the make system somehow?

Reply:
not sure, I saw lots of "internal" environment stuff that might be a problem security wise to be exposed - so we need to be careful there.

Reply:
OK, I'll try to figure something out.

Reply:
OK, I added an number of env vars that I thought would be useful.  I hope this is still safe.